packages/tokens/src/Tokens.stories.tsx

```diff
@@ -85,6 +84,39 @@ function SpacingBlock({ space }: { space: keyof typeof spacing }) {
   );
 }
 
+export const Shadow = () => (
+  <div
+    className={css`
+      display: flex;
+      gap: ${spacing[400]}px;
+    `}
+  >
+    <div
+      className={css`
+        box-shadow: ${shadow.light[100]};
+        background-color: ${color.light.background.primary.default};
+        padding: ${spacing[1600]}px;
+        border-radius: ${borderRadius[200]}px;
+        color: ${color.light.text.primary.default};
+      `}
+    >
+      shadow.light[100]
+    </div>
+
+    <div
+      className={css`
+        box-shadow: ${shadow.dark[100]};
+        background-color: ${color.dark.background.primary.default};
+        padding: ${spacing[1600]}px;
+        border-radius: ${borderRadius[200]}px;
+        color: ${color.dark.text.primary.default};
+      `}
+    >
+      shadow.dark[100]
+    </div>
```

Comment:
can we set up this token on a dark background similar to what we did for the colors story?

Reply:
what do you mean?

Reply:
![Screenshot 2024-07-15 at 2 47 29 PM](https://github.com/user-attachments/assets/f01113e4-f399-4ad7-8b3f-e070ca55e2a6)
![Screenshot 2024-07-15 at 2 47 18 PM](https://github.com/user-attachments/assets/53a523af-e569-4ad6-bdbb-ebabfd37a2da)